src/routes/(protected)/calendar/+page.svelte

```
@@ -158,77 +180,59 @@
+        s.border,
+        s.bg,
+        s.hover,
+        compact && s.compact,
+        item.last_state === "Completed" && "line-through opacity-60",
+        href && "no-underline"
+    )}
+    {@const title = compact
+        ? `${item.show_title}${item.season ? ` S${item.season}E${item.episode}` : ""}`
+        : undefined}
+    {#if href}
+        <a {href} class={classes} {title}>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)